pastepwn/core/pastepwn.py

```diff
@@ -94,7 +94,10 @@ def start(self):
         if self.__exception_event.is_set():
             self.logger.error("An exception occured. Aborting the start of PastePwn!")
             exit(1)
-
+        if len(self.scraping_handler.scrapers) == 0:
+            from pastepwn.scraping.pastebin import PastebinScraper
```

Comment:
Hi there. I changed it a bit so that it will only import the module when necessary. This also circumvents the circular import.

Reply:
I got this idea too, but I thought it was a bad practice.